website/addons/wiki/templates/edit.mako

```diff
@@ -219,7 +219,7 @@
   <div class="modal-dialog">
     <div class="modal-content">
       <div class="modal-header">
-        <h3 class="modal-title">The permissions for this page have changed</h3>
```

Comment:
@geeksnglitter @sloria There is no clear rule for this in style guide. But I think for the long sentence, it may be better to just make the first word uppercase.  Because in general, uppper case is not so readable anyway, compared with lower case.